docs/maui/views/MediaElement.md

```diff
@@ -67,10 +67,11 @@ To access the `MediaElement` functionality, the following platform specific setu
 <!-- markdownlint-disable MD025 -->
 ### [Android](#tab/android)
 
-To initialize the `MediaElement` on Android, the `LaunchMode` of the applications `Activity` must be set to `LaunchMode.SingleTask` as per the following example.
+To initialize the `MediaElement` on Android, the `LaunchMode` of the applications `Activity` must be set to `LaunchMode.SingleTask` and you must add 'ResizableActivity=true` as per the following
```

Comment:
Thanks James! 

One small nit - it looks like an apostrophe got used instead of a backtick

```suggestion
To initialize the `MediaElement` on Android, the `LaunchMode` of the applications `Activity` must be set to `LaunchMode.SingleTask` and you must add `ResizableActivity=true` as per the following
```